etc/rsync.profile

```diff
@@ -0,0 +1,54 @@
+# Firejail profile for rsync
+# Description: a fast, versatile, remote (and local) file-copying tool
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include rsync.local
+# Persistent global definitions
+include globals.local
+
+# This profile is focused on using rsync as an client for downlaoding, it is not writen for using rsync
+#  - as an daemon (rsync --daemon)
+#  - creating backup of $HOME or a full system
+
+blacklist /tmp/.X11-unix
```

Comment:
maybe `x11 none`?

Reply:
since `x11 none` needs `net none` to work it isn't possible here, but thanks for keeping an eye on that.